airsonic-main/src/main/java/org/airsonic/player/repository/AlbumRepository.java

```diff
@@ -43,6 +43,7 @@ public interface AlbumRepository extends JpaRepository<Album, Integer> {
 
     public Optional<Album> findByIdAndStarredAlbumsUsername(Integer id, String username);
 
+    @Transactional
     public void deleteAllByPresentFalse();
```

Comment:
Cleaning up missing files wasn't working so I had to add the `@Transactional` here to make that work. I was merely mimicking the other `deleteAll*` methods in that path. I'm no hibernate expert.